recipes/pvWebMonitor/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pvWebMonitor" %}
+{% set version = "2021.0.0rc1" %}
```

Comment:
Is there going to be a release soon? Normally we don't want to have pre-release versions on the main label.

Reply:
@xhochy :  Thanks for the prompt review!  Now that I know the `meta.yaml` recipe results in a successful build, a new, regular release can be posted and this PR revised to target that release.  Is that preferable?